tests/page-objects/contacts/contacts.po.js

```diff
@@ -94,7 +104,7 @@ module.exports = {
   search: async query => {
     searchBox.clear();
     searchBox.sendKeys(query);
```

Comment:
Both [clear](https://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.clear) and [sendKeys](https://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.sendKeys) look to return promises and you'll need to `await` them to be sure those keystrokes happen before the `click` on the following line. With that though, I would have expected this test to fail most or all of the time, so perhaps I'm missing something here. Is there something else involved that's causing these actions to happen in the desired sequence?
```suggestion
    await searchBox.clear();
    await searchBox.sendKeys(query);
```